src/animation/AnimationAction.js

```diff
@@ -11,11 +11,12 @@ import { WrapAroundEnding, ZeroCurvatureEnding, ZeroSlopeEnding, LoopPingPong, L
  *
  */
 
-function AnimationAction( mixer, clip, localRoot ) {
+function AnimationAction( mixer, clip, localRoot, isAdditive = false ) {
```

Comment:
How about making `AdditiveAnimationAction` and extend `AnimationAction` instead?

Reply:
I think that this could be a mode rather than a class.  Similar to Pixel Blending modes.  The default is Over ( source * a +  dest * (1 - a) ), and this is Additive ( source * a + dest * 1 ).  I wouldn't be surprised if there are additional animation modes.

Reply:
Unity calls this Blending mode and it is an enum type, rather than a true/false.

- "On each layer, you can specify the mask (the part of the animated model on which the animation would be applied), and the Blending type. Override means information from other layers will be ignored, while Additive means that the animation will be added on top of previous layers."  (Source: https://docs.unity3d.com/Manual/AnimationLayers.html)

In the Porter Duff compositing modes we get:
- SRC - replace.
- SRC_OVER - is override.
- ADD is additive.

Reply:
So I could introduce a new enum called AnimationBlendMode containing Over and Additive values, then change that parameter to be called 'blendMode' and default it to 'Over'. Does that sound about right?

Reply:
Blending mode sounds good to me. I was just trying to avoid ending in a situation like this:

```js
function AnimationAction( mixer, clip, localRoot, isAdditive = false, isSubtrative = true )
```

Can we reuse already available constants?

```js
function AnimationAction( mixer, clip, localRoot, blending = NoBlending | AdditiveBlending )
```

Reply:
>Can we reuse already available constants?

Please, lets not do that. Instead, define something like:
```js
NormalAnimationBlending = 3300; // or 0
AdditiveAnimationBlending = 3301; // or 1
OverrideAnimationBlending = 3302; // or 2 -- if we support that
```